cMake/FreeCAD_Helpers/SetupQt.cmake

```diff
@@ -1,38 +1,44 @@
 # -------------------------------- Qt --------------------------------
 
-find_package(Qt5Core REQUIRED)
-set(CMAKE_AUTOMOC TRUE)
-
-# For FreeCAD 0.20, pegged to Ubuntu 18.04 LTS:
-if(${Qt5Core_VERSION} VERSION_LESS "5.9")
-    message (FATAL_ERROR "FreeCAD v0.20 requires Qt5 5.9 or later")
+set(FREECAD_QT_COMPONENTS Core Network Xml Concurrent)
```

Comment:
Just a minor thing and possibly a personal preference.
Sorting the components alphabetically, makes it easier to find a specific one, especially if a list is longer than this one.

Reply:
Probably a good idea (especially since conveniently "Core" comes first alphabetically, and having it first somehow feels right).